tests/lib/rules/jsx-indent.js

```diff
@@ -159,6 +188,15 @@ ruleTester.run('jsx-indent', rule, {
       '  );',
       '}'
     ].join('\n'),
+    output: [
+      'function App() {',
+      '  return (',
+      '    <App>',
+      '  <Foo />',
+      '</App>',
+      '  );',
+      '}'
+    ].join('\n'),
```

Comment:
@ljharb a fix included in this PR or a fix elsewhere? 

@CiGit had said this:

> To have the output you (and I) expect, the way errors are computed should be changed. ie JSXElement's indentation compared to something like ROOT - 1line, not to its direct parent / opening tag.
> 
> IMO that's an other issue

If I understand correctly, this will require detection logic changes. I unfortunately don't have cycles to dig that deep into the internals at the moment. If anyone else does, feel free to continue--I don't need commit "credit" hehe